front-end/tailwind.config.js

```diff
@@ -17,9 +17,9 @@ module.exports = {
         300: "#374151",
       },
       "green": {
-        100: "#A4BCB9",
-        200: "#3D5245",
-        300: "#C9D2D1"
+        100: "#C9D2D1",
+        200: "#A4BCB9",
+        300: "#3D5245"
```

Comment:
Is this intentional?

Reply:
Yes - was it correct to have changed it? I thought from 100 to 300 should be lightest to darkest, and it was out of order before I changed it. What used to be 300 and is now 100 wasn't ever used in the file before, and I made sure all the other uses of the 100/200 were properly updated.

Reply:
You're definitely right on ordering from light to dark. Just you didn't make CSS changes so I wasn't sure if it was intentional. 👍